src/states/Lobby.gd

```diff
@@ -38,25 +38,34 @@ onready var start_button := $"%StartButton" as Button
 var chosen_colors := {}
 # Dictionary from player_id to row in the table.
 var player_id_to_row := {}
+# Dictionary from player_id to ping, in ms.
+var player_id_to_ping := {}
 
 
 # Called when the node enters the scene tree for the first time.
 func _ready() -> void:
 	if 1 in Multiplayer.player_info:
 		server_name.text = Multiplayer.player_info[1].name + "'s Server"
+	if get_tree().is_network_server():
+		Multiplayer.player_latency[1] = 0
+		player_id_to_ping = Multiplayer.player_latency
 	generate_button_grid()
 	update_table()
+	var error = Multiplayer.connect("latency_updated", self, "on_latency_update")
+	assert(not error)
```

Comment:
Do we want to just edit the project configuration to ignore these warnings?

Reply:
![image](https://user-images.githubusercontent.com/22134557/210920465-932bd837-2a88-4c12-b0e9-c7b48cf7eb0c.png)